assessmentModel/src/commonMain/kotlin/serialization/ModuleInfoObject.kt

```diff
@@ -21,12 +21,12 @@ val moduleInfoSerializersModule = SerializersModule {
 data class ModuleInfoObject(
     override val assessments: List<TransformableAssessment>,
     override var packageName: String? = null,
-    override val bundleIdentifier: String? = null): ResourceInfo, EmbeddedJsonModuleInfo {
+    override val bundleIdentifier: String? = null,
+    @Transient
+    override val jsonCoder: Json = Serialization.JsonCoder.default): ResourceInfo, EmbeddedJsonModuleInfo {
```

Comment:
So... the idea was that the module info objects could be deserialized with a "type" that is registered with the `moduleInfoSerializersModule` where each module would define it's own type. If you wanted to instantiate this thing directly and *not* use the default coder, then would probably be better to be explicit and have a *different* class such as:

```
class ModuleInfoImpl(
     override val assessments: List<TransformableAssessment>,
     override var resourceInfo: ResourceInfo,
     override val jsonCoder: Json
) : EmbeddedJsonModuleInfo 
```

Reply:
I went ahead and changed the sample app to load modules from a JSON file.